newrelic/newrelic.go

```diff
@@ -1,3 +1,8 @@
+fmt.Print("\n\n **************************** \n")
```

Comment:
Probably remove these once you're done testing.

Reply:
Interesting, I didn't add those lines, perhaps they came in from another unmerged PR?

Reply:
fixed @zlesnr